src/craftdef.cpp

```diff
@@ -907,8 +907,18 @@ class CCraftDefManager: public IWritableCraftDefManager
 					<< " against " << def->dump() << std::endl;*/
 
 				if (def->check(input, gamedef)) {
+					// Check if the crafted node/item exists
+					CraftOutput out = def->getOutput(input, gamedef);
+					ItemStack is;
+					is.deSerialize(out.item, gamedef->idef());
+					if (!is.isKnown(gamedef->idef())) {
+						errorstream << "trying to craft nonexistent " << out.item << ", ignoring recipe" << std::endl;
```

Comment:
This is not important enough to be an error. If this even should be logged, write it to `infostream`.

Reply:
I believe it should be logged to aid mod development, moving it to infostream

Reply:
Sorry, forgot to mention. The line is too long. Please split it to keep it within the character limit (80-90).